config/config.exs

```diff
@@ -26,4 +26,6 @@ use Mix.Config
 # Configuration from the imported file will override the ones defined
 # here (which is why it is important to import them last).
 #
+config :sparrow, Sparrow.PoolsWarden,
+%{enabled: true}
```

Comment:
Maybe pass it to Application start function (eg. as optional parameter ) instead of env?

Reply:
Why? Currently whole configuration is done by config files and in Aplication start function FCM and APNS configs are extracted from environment variables as well.

Reply:
ok